bugwarrior/docs/services/redmine.rst

```diff
@@ -28,6 +28,22 @@ If you want to ignore verifying the SSL certificate, set::
 
     redmine.verify_ssl = False
 
+Specify the parameters to filter issues
++++++++++++++++++++++++++++++++++++++++
+
+If you need to take control over the filtering of issues, you may use the
+`redmine.query` setting.
+
+Be aware though that the `redmine.issue_limit` and `redmine.only_if_assigned`
+settings will be then be ignored.
+
+You'll have to include their redmine equivalent in the query::
+
+    redmine.query = project_id=10&status_id=15&limit=100&assigned_to_id=me
```

Comment:
Sorry if I misled you regarding the extent to which `query` should override default behavior -- I hadn't realized how much simpler the Redmine service is than the others in which we've implemented this option. In this case I don't think it should override either of these configuration options and therefore you could just add it to the conventional querystring as I believe you first suggested.

Reply:
I handled it the way I first imagined it.